src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+
+}
+
+.bi {
+  width: calc(100% - 62px);
```

Comment:
Strange solution, instead of width calculating you can set paddings to parent container